### DISCUSSION TITLE
Question: [CalendarDatePicker] Why does the Month name in flyout has extra prefix and suffix characters?

### DISCUSSION THREAD

**Discussion by almedina-ms** [microsoft/microsoft-ui-xaml#7458] [Q&A]
When using the Microsoft.Windows.Apps.Test framework to interact with the CalendarDatePicker flyout control, the 'HeaderButton' (the button that contains the Month and year) has an Automation Name value that contains extra leading and ending characters; the characters have a value of '8206' which is a zero-width character. Is there a reason why this character appears? Is there some way to avoid those characters from being in the Automation Name?

**Comment by StephenLPeters** [comment]
Very strange, how does this end up affecting the narrator experience?

**Comment by almedina-ms** [comment]
It may not modify the narrator experience but for automated testing it yields unexpected results. Turns out the 8206 character appended is a LTR delimiter (https://en.wikipedia.org/wiki/Left-to-right_mark), I haven't seen other controls appending that character

**Comment by StephenLPeters** [comment]
Interesting, the Month is a localized string, my guess is its included in there?  Is this causing you issues?
